### PR TITLE
[FEATURE] Added direct link to icon set

### PR DESCRIPTION
**Pull request by wolv-dev**
If pull request https://github.com/shopware/devdocs/pull/389 will be declined due to duplicate content I made this one.

It's easier to find the icon set and link will be added to search machine index.

### CODE REVIEW

**Comment by mitelg** [comment]
hey @wolv-dev any updates here?

File: source/designers-guide/index.html [hunk 1]
```diff
@@ -25,6 +25,7 @@
 <ul class="guide-list">
     <li><a href="{{ site.url}}/designers-guide/javascript-coding-style/">JavaScript coding style</a></li>
     <li><a href="{{ site.url }}/styletile" title="Shopware UI components overview">Shopware UI components overview</a></li>
+    <li><a href="{{ site.url }}/styletile/_components-icon-set.html" title="Frontend icon set overview">Frontend icon set overview</a></li>
```
[the remark this answers is not in the document]
Comment (wolv-dev) [reply source/designers-guide/index.html]: I added the link

**Comment by wolv-dev** [comment]
@mitelg Updated as well.

**Comment by mitelg** [comment]
thanks for your contribution @wolv-dev :+1: :tada:

**Pull request merged**